sdk/metric/reader.go

```diff
@@ -117,9 +117,20 @@ type ReaderOption interface {
 	PeriodicReaderOption
 }
 
-// WithTemporality uses the selector to determine the Temporality measurements
-// from instrument should be recorded with.
-func WithTemporality(selector func(instrument InstrumentKind) Temporality) ReaderOption {
+// TemporalitySelector selects the temporality to use based on the InstrumentKind.
+type TemporalitySelector func(InstrumentKind) Temporality
+
+// DefaultTemporalitySelector is the default TemporalitySelector used if
+// WithTemporalitySelector is not provided. CumulativeTemporality will be used
+// for all instrument kinds if this TemporalitySelector is used.
+func DefaultTemporalitySelector(InstrumentKind) Temporality {
```

Comment:
nit: does this need to be exported?  Since it is the default, users don't need to ever set it.

Reply:
Ah, got it.  Its fine to leave it exported.